ra/ra.go

```diff
@@ -1878,7 +1887,12 @@ func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.New
 	// If new authorizations are needed, call AddPendingAuthorizations. Also check
 	// whether the newly created pending authz's have an expiry lower than minExpiry
 	if len(newAuthzs) > 0 {
-		authzIDs, err := ra.SA.AddPendingAuthorizations(ctx, &sapb.AddPendingAuthorizationsRequest{Authz: newAuthzs})
+		var authzIDs *sapb.AuthorizationIDs
+		if features.Enabled(features.NewAuthorizationSchema) {
+			authzIDs, err = ra.SA.NewAuthorizations(ctx, &sapb.AddPendingAuthorizationsRequest{Authz: newAuthzs})
```

Comment:
minor nit: It looks like you could construct one `*sapb.AddPendingAuthorizationsReqest` outside of the `if` statement alongside `authzIDs` and then simplify both clause bodies to use it as the 2nd arg to the relevant RPC.